src/ballet/shred/fd_shred.h

```diff
@@ -162,12 +181,12 @@ struct __attribute__((packed)) fd_shred {
       /* Bit field (MSB first)
          See FD_SHRED_DATA_FLAG_*
 
-          [XX.. ....] Block complete?       0b00=no 0b11=yes (implies Entry batch complete)
+          [XX.. ....] Block complete?       0b00=no 0b01=no 0b11=yes (implies Entry batch complete)
           [.X.. ....] Entry batch complete?  0b0=no  0b1=yes
           [..XX XXXX] Reference tick number */
       /* 0x55 */ uchar  flags;
 
-      /* Actual shred size including headers and Merkle proof */
+      /* Actual shred size including headers, Merkle proof and previous Merkle root if chained */
```

Comment:
I am pretty sure this is not right, though it's also wrong on main. The shred spec, the comments in `fd_shred_parse`, and `fd_shred_payload_sz` all seem to claim it is just headers+payload

Reply:
You're right, thanks! Corrected.